pom.xml

```diff
@@ -214,6 +216,22 @@
         <version>${jboss-log-manager.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-api</artifactId>
+        <version>${opentelemetry.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-sdk</artifactId>
+        <version>${opentelemetry.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-exporters-inmemory</artifactId>
```

Comment:
For my own understanding, it is a regular dependency or a test dependency?

Reply:
Testing only